content/src/view.js

```diff
@@ -245,16 +250,21 @@ function initialPaneState() {
 }
 
 function setOnCallback(tag, cb) {
-  state.callbacks[tag] = cb;
+  if (state.callbacks[tag] == null){
+    state.callbacks[tag] = [];
+  }
+  state.callbacks[tag].push(cb);
 }
 
 function fireEvent(tag, args) {
   if (tag in state.callbacks) {
-    var cb = state.callbacks[tag];
-    if (cb) {
-      cb.apply(null, args);
-    }
-  }
+    var cbs = state.callbacks[tag].slice();
```

Comment:
Actually I'm thinking, since we are not altering the array, do we need that `.slice()`? Is there any reason for that?